### PR TITLE
LPS-199121 + LPS-197719 - Enable Result Ranking scoping by default (beta) + Update related wording

### PR DESCRIPTION
**Pull request by oliv-yu**
~~https://liferay.atlassian.net/browse/LPS-189239 - Update management toolbar with filter + search controls~~ (Merged in https://github.com/brianchandotcom/liferay-portal/pull/145912)
https://liferay.atlassian.net/browse/LPS-199121 - FF: From Dev to Beta | Result Ranking scoping becomes enabled by default featuring Everything, Site (Beta) and Blueprints (Beta) scopes
https://liferay.atlassian.net/browse/LPS-197719 - Update wording for the error messages when creating duplicate RR entries

Sending with management toolbar updates to avoid merge conflicts 🙏 

Updates:
- Removal of FF LPS-159650 and LPS-157988
- Adding beta badge to the dropdown option
- Scope column on the result rankings table + scope info on the result ranking page is always visible
- Updated wording, so references to active search queries includes the mention of scope

CDN of the portal-search-tunings-rankings-web JSON was updated, so the "Learn More" link should be available!

### Management Toolbar Screenshots
Screenshot 2023-12-21 at 10 14 53 AM
Screenshot 2023-12-21 at 10 15 18 AM
Screenshot 2023-12-21 at 10 15 38 AM

### New Wording
Screenshot 2023-12-20 at 4 40 44 PM
Screenshot 2023-12-20 at 4 41 06 PM
Screenshot 2023-12-20 at 4 59 46 PM

Rebased on top of https://github.com/liferay-search/liferay-portal/pull/1196 to avoid merge conflicts

### CODE REVIEW

File: modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/js/components/ResultRankingsAdd.es.js [hunk 1]
```diff
@@ -206,6 +209,14 @@ function ResultRankingsAdd({cancelURL, fetchSitesURL, formName}) {
 								<div className="autofit-col-expand">
 									<div className="list-group-text text-dark">
 										{SCOPE_INFO[item].label}
+
+										{SCOPE_INFO[item].showBetaBadge && (
+											<ClayBadge
+												displayType="info"
+												label="beta"
+												translucent
+											/>
```
Comment (thektan) [inline modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/js/components/ResultRankingsAdd.es.js:218]: Can you try using the BetaBadge component from `frontend-js-components-web`?

```
import {BetaBadge} from 'frontend-js-components-web';
```

Also 2 things about the styling:
1. Can you vertically center the beta badge and give it maybe 4px or 8px padding between the text and badge? It looks really misaligned in the screenshot.
Screenshot 2023-12-21 at 11 28 35 AM

2. Did the styling for the beta badge change? From the original beta badge docs the text is all caps. https://liferay.atlassian.net/wiki/spaces/PEDS/pages/2312110481/Identifying+BETA+Features+in+the+product
Screenshot 2023-12-21 at 11 47 45 AM

**Review by thektan — CHANGES_REQUESTED** [review]
Just 1 requested change about the beta badge.

Also are these backend commits sent separately to BChan? There's a lot of commits in this PR so BChan would highly reject it as-is.

File: modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/js/components/ResultRankingsAdd.es.js (hunk 1 above)
Comment (oliv-yu) [reply modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/js/components/ResultRankingsAdd.es.js:218]: @thektan Sure! I didn't know there was a Beta Badge already, I imported it but the badge is a bit bigger than the ClayBadge, also it doesn't look like it takes in additional styling? I was wondering how to fix the alignment that way.

Screenshot 2023-12-21 at 12 57 57 PM

For the beta case, I'm actually not sure what is the standard, I did a search and it looked like several JS components around portal that kept the label "beta" lower case. It looks like the frontend BetaBadge is just capitalized, too 🤔 ... should we change to all caps?

**Comment by oliv-yu** [comment]
> Also are these backend commits sent separately to BChan? There's a lot of commits in this PR so BChan would highly-likely reject it as-is.

Yup! Those commits are in https://github.com/liferay-search/liferay-portal/pull/1196

File: modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/js/components/ResultRankingsAdd.es.js (hunk 1 above)
Comment (thektan) [reply modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/js/components/ResultRankingsAdd.es.js:218]: You can center doing something like (but make it scoped to the ClayDropDown.Menu)

```css
.list-group-text {
  align-items: center;
  display: flex;
}
```

In that case, we can continue to use `ClayBadge` instead, but let's uppercase using `className="text-uppercase"` too.

Comment (oliv-yu) [reply modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/js/components/ResultRankingsAdd.es.js]: Added the updates! Lmk if that looks ok 👍  
Screenshot 2023-12-21 at 3 34 13 PM

Comment (thektan) [reply modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/js/components/ResultRankingsAdd.es.js]: Yup much better! Thanks! 💯

**Comment by BryanEngler** [comment]
@oliv-yu @thektan would any changes from https://github.com/brianchandotcom/liferay-portal/pull/145590 be applicable here?

**Comment by oliv-yu** [comment]
ci:test:sf

**Comment by oliv-yu** [comment]
ci:test:search

**Comment by oliv-yu** [comment]
ci:test:search

**Comment by oliv-yu** [comment]
ci:test:relevant

**Comment by oliv-yu** [comment]
@BryanEngler I'm not exactly sure what's going on, but it looks like there's some test failures occurring with result rankings pinning/hiding. I was trying on master and enabled FFs 157988 + 159650 but the pinning/hiding didn't work for me? This PR removes the FF so it's suddenly giving all these results 😅

**Comment by BryanEngler** [comment]
testing being done here https://github.com/xbrianlee/liferay-portal/pull/1472

**Comment by xbrianlee** [comment]
Manually forwarded
https://github.com/brianchandotcom/liferay-portal/pull/145988

**Pull request closed without merging**